compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/util/Flags.java

```diff
@@ -81,6 +81,7 @@ public class Flags {
     public static final long INFER = ANY_FUNCTION << 1;                         //  40
     public static final long ENUM_MEMBER = INFER << 1;                          //  41
     public static final long QUERY_LAMBDA = ENUM_MEMBER << 1;                   //  42
+    public static final long TYPE_DEF = QUERY_LAMBDA << 1;                         //  43
```

Comment:
```suggestion
    public static final long TYPE_DEF_CREATED= QUERY_LAMBDA << 1;                         //  43
```

May be?

Reply:
+1, would be more meaningful.

Reply:
+1

Reply:
Updated [11da802](https://github.com/ballerina-platform/ballerina-lang/pull/40678/commits/11da8025f891454affd08a73696b7a0b8babb281)